Sources/TuistCore/Models/TargetPattern.swift

```diff
@@ -0,0 +1,16 @@
+/// Patterns for matching against a target.
+public enum TargetPattern: Hashable {
```

Comment:
What about this instead?

```suggestion
public enum TargetQuery: Hashable {
```